src/__test__/components/data-exploration/hierarchical-tree/HierarchicalTree.test.jsx

```diff
@@ -1,26 +1,51 @@
 import React from 'react';
 import { mount } from 'enzyme';
-import HierarchicalTree from '../../../../components/data-exploration/hierarchical-tree/HierarchicalTree';
-import waitForComponentToPaint from '../../../../utils/tests/waitForComponentToPaint';
+
+import {
+  EditOutlined, DeleteOutlined, // , CheckOutlined, CloseOutlined,
+} from '@ant-design/icons';
+
+import HierarchicalTree from 'components/data-exploration/hierarchical-tree/HierarchicalTree';
+import waitForComponentToPaint from 'utils/tests/waitForComponentToPaint';
+
 import '__test__/test-utils/setupTests';
 
 const experimentId = 'asd';
```

Comment:
can you import `fake` from `__test__/test-utils/constants` and use `fake.EXPERIMENTT_ID` instead? It's closer to the real data, so that would be good.